Custom-Video-Driver/Lets-Build-OTPublisher/Multi-Cam-Capturer/TBExampleMultiCamCapture.m

```diff
@@ -432,6 +432,23 @@ - (void)noFramesTimerFired:(NSTimer *)timer {
     }
 }
 
+- (void) setTorchMode:(AVCaptureTorchMode) torchMode {
+    
+    AVCaptureDevice *device = [[self videoInput] device];
```

Comment:
@IGitGotIt this code change is needed only on capturer and not in the sdk?

Reply:
Yes, it  is just localized to one sample app capturer. No SDK changes.